src/DependencyInjection/M6WebGuzzleHttpExtension.php

```diff
@@ -158,19 +158,21 @@ protected function setGuzzleProxyHandler(ContainerBuilder $container, $clientId,
         $curlMultihandler->addMethodCall('setDebug', [$container->getParameter('kernel.debug')]);
 
         if (array_key_exists('guzzlehttp_cache', $config)) {
-            $defaultTtl = $config['guzzlehttp_cache']['default_ttl'];
-            $headerTtl = $config['guzzlehttp_cache']['use_header_ttl'];
-            $cacheServerErrors = $config['guzzlehttp_cache']['cache_server_errors'];
-            $cacheClientErrors = $config['guzzlehttp_cache']['cache_client_errors'];
             if (is_null($cacheService = $this->getServiceReference($container, $config['guzzlehttp_cache']['service']))) {
                 throw new \InvalidArgumentException(sprintf(
                     '"guzzlehttp_cache.service" requires a valid service reference, "%s" given',
                     $config['guzzlehttp_cache']['service']
                 ));
             }
 
-            $curlhandler->addMethodCall('setCache', [$cacheService, $defaultTtl, $headerTtl, $cacheServerErrors, $cacheClientErrors]);
-            $curlMultihandler->addMethodCall('setCache', [$cacheService, $defaultTtl, $headerTtl, $cacheServerErrors, $cacheClientErrors]);
+            $defaultTtl = $config['guzzlehttp_cache']['default_ttl'];
+            $headerTtl = $config['guzzlehttp_cache']['use_header_ttl'];
+            $cacheServerErrors = $config['guzzlehttp_cache']['cache_server_errors'];
+            $cacheClientErrors = $config['guzzlehttp_cache']['cache_client_errors'];
+            $ignoreCacheErrors = $config['guzzlehttp_cache']['ignore_cache_errors'] ?? false;
```

Comment:
Why "?? : false" - The default value should assure there is always true or false, no ?